### DISCUSSION TITLE
Ecommerce Landing Page with AdminLTE

### DISCUSSION THREAD

**Discussion by noblemfd** [ColorlibHQ/AdminLTE#3637] [Q&A] [answered]
I am using AdminLTE-3 for an E-commerce application.

I have two sides: 1. Admin Backend and 2. Landing Page.

Where I have issue is the Landing Page.

Is there a way to use AdminLTE-3 for the Landing Page (A beautiful Landing Page)? Any clue, info or link?

I want to use ADMINLTE-3

Thanks

**Comment by danny007in** [comment, marked as answer]
For landing use different theme
and
we will add landing page in AdminLTE v4

**Reply by noblemfd** [reply]
When is AdminLTE v4 going to be released

**Reply by danny007in** [reply]
keep tracking on https://github.com/ColorlibHQ/AdminLTE/projects/13

**Reply by danny007in** [reply]
> 
> 
> keep tracking on [ColorlibHQ/AdminLTE/projects/13](https://github.com/ColorlibHQ/AdminLTE/projects/13)

propably 
after 3 months

**Comment by REJack** [comment]
Overall you can use boxed or default layout without sidebar voila you have a simple „landing“ page.
